pkg/worker/analyzed_audit_job_test.go

```diff
@@ -109,6 +128,49 @@ func (suite *WorkerSuite) TestAnalyzedAuditJob() {
 	_, paymentErr := resolvers.PlanPaymentsUpdate(worker.Logger, worker.Store, payment.ID, reviewChanges, suite.testConfigs.Principal)
 	suite.NoError(paymentErr)
 
+	mockEmailTemplateService.
+		EXPECT().
+		GetEmailTemplate(gomock.Eq(email.DataExchangeApproachMarkedCompleteTemplateName)).
+		Return(testTemplate, nil).
+		AnyTimes()
+
+	mockEmailService.
+		EXPECT().
+		GetConfig().
+		Return(&emailtestconfigs.TestEmailServiceConfig).
+		AnyTimes()
+
+	mockEmailService.
+		EXPECT().
+		Send(
+			gomock.Any(),
+			gomock.Any(),
+			gomock.Any(),
+			gomock.Eq(expectedSubject),
+			gomock.Any(),
+			gomock.Eq(expectedBody),
+			gomock.Any(),
+		).AnyTimes()
+
+	// Update Plan Data Exchange Approach
+	dea, deaErr := resolvers.PlanDataExchangeApproachUpdate(
```

Comment:
I think you are missing the assertion that the analyzed audit contains a marked as complete entry.

Reply:
The thinking here is not to assert the form of data, which is done elsewhere. Rather this is about flow assertion to ensure that the analyzed audit job is triggering downstream behavior. Clay had a similar question about whether DEA tests are appropriate here. My answer is no, not really, but we don't currently have the ability to make mock flow assertions about the analyzed audit job so this is the best we can do.